pkgs/top-level/all-packages.nix

```diff
@@ -1884,6 +1884,8 @@ with pkgs;
 
   immich-cli = callPackage ../tools/misc/immich-cli { };
 
+  inshellisense = callPackage ../tools/misc/inshellisense { };
```

Comment:
Shall be moved to `pkg/by-name`, otherwise LGTM.

Reply:
Done!